content/en/_index.md

```diff
@@ -197,6 +233,42 @@ public static void listInfoTypes() throws IOException {
   }
 }
 {{< /tab >}}
+{{< tab header="PHP" >}}
+/**
+ * Lists all Info Types for the Data Loss Prevention (DLP) API.
+ */
+use Google\Cloud\Dlp\V2\DlpServiceClient;
+
+/**
+ * Lists all Info Types for the Data Loss Prevention (DLP) API.
+ *
+ * @param string $filter        (Optional) filter to use
+ * @param string $languageCode  (Optional) language code, empty for 'en-US'
+ */
+function list_info_types(string $filter = '', string $languageCode = ''): void
```

Comment:
If these are optional, should they have filler values?

Reply:
They are made optional by having default values in the function signature.

Reply:
I understand that. My question was should we have better defaults than empty strings?

Reply:
Do we want to remove these as parameters and hard-code the values for purposes of the guide?

Reply:
In the other samples, we tend to set some default and a comment telling users to set these;

```
function list_info_types(
    // TODO(developer): Replace these values before running your sample: 
    string $filter = 'your-filter', 
    string $languageCode = 'your-language-code',
):
```

Reply:
I will make that change. It's not what the Java variant we're showing here does.

Reply:
Java doesn't have the optional types, which is why we use the header function. The equivalent is: 

```java
public static void main(String args...){
    // TODO(developer): Replace these values before running your sample: 
    String filter = "your-filter";
    String languageCode = "your-language-code";
    exampleSnippet(filter, langaugeCode);
}
```

The alternative is comments, which I don't like because they don't do any lint of compile checks:
```java
    // TODO(developer): Replace these values before running your sample: 
    // String filter = "your-filter";
    // String languageCode = "your-language-code";
```

So historically, there have been many examples of comments drifting over time when samples get updated.